.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: coq / main
+
+on:
+  push:
+    branches: [master, main]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+      
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Pull coq docker image
+        run: docker pull coqorg/coq:8.9
+
+      - name: Run exercism/coq ci (runs tests) for all exercises
+        run: |
+          export NJOBS=2
+          # Disable for now, as this step was already disabled in Travis config
+          # docker run --rm coqorg/coq:8.9 coq_makefile -f _CoqProject -o Makefile && make && make install
```

Comment:
I don't know for sure, but the docs seem to imply that you could just directly specify the Docker image to run on: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer